handler/list.go

```diff
@@ -0,0 +1,33 @@
+package handler
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/cloudradar-monitoring/plexus/api"
+)
+
+// ListSession godoc
+// @Summary Lists all active session.
+// @Tags session
+// @Produce  application/json
+// @Success 200 [{object}] api.Session
+// @Failure 400 {object} api.Error
+// @Failure 500 {object} api.Error
+// @Failure 502 {object} api.Error
+// @Router /session [get]
```

Comment:
It's `{array}` instead of `[{object}]`
```suggestion
// ListSession godoc
// @Summary Lists all active session.
// @Tags session
// @Produce  application/json
// @Success 200 {array} api.Session
// @Failure 400 {object} api.Error
// @Failure 401 {object} api.Error
// @Failure 500 {object} api.Error
// @Router /session [get]
```